api/authenticator.py

```diff
@@ -166,8 +167,8 @@ def create_authentication_document(self):
                 links[rel] = dict(href=value, type="text/html")
 
         doc = OPDSAuthenticationDocument.fill_in(
-            base_opds_document, auth_type, "Library", opds_id, None, "Barcode",
-            "PIN", links=links
+            base_opds_document, auth_type, _("Library"), opds_id, None, _("Barcode"),
+            _("PIN"), links=links
```

Comment:
I'm a little worried about this because individual libraries will want to modify this sort of vocabulary, but we _also_ want it to be localized. I don't see a better solution for now, though.